README.md

```diff
@@ -56,7 +81,7 @@ You may also install dependencies for only a subset of the packages
 pnpm install -w --filter <package>...
 ```
 
-### Launching
+### Launching for development
```

Comment:
Add a link to rmf demos setup.

Reply:
https://github.com/open-rmf/rmf-web/pull/994/commits/31b6489a5c6b69a38900f3fb52dfdff5d6a43eaa